app/org/sagebionetworks/bridge/dynamodb/DynamoScheduledActivityDao.java

```diff
@@ -117,7 +94,12 @@ final void setDdbMapper(DynamoDBMapper mapper) {
             query.withExclusiveStartKey(map);
         }
         
-        QueryResultPage<DynamoScheduledActivity> queryResult = mapper.queryPage(DynamoScheduledActivity.class, query);
+        QueryResultPage<DynamoScheduledActivity> queryResult = null;
+        try {
+            queryResult = mapper.queryPage(DynamoScheduledActivity.class, query);
+        } catch(AmazonServiceException e) {
+            throw new BadRequestException(e.getErrorMessage());
+        }
```

Comment:
AmazonServiceException includes things like internal server error and throttling exceptions. We shouldn't munge them all into a 400 Bad Request exception, as that gives the client the wrong impression of why the server call failed. Additionally, throwing Bad Request exceptions hides the exception from our ExceptionInterceptor, which means we lose visibility into when our dependent services are behaving badly.

AmazonServiceException has a getStatusCode() field. If it's in the 4XX range, we can return a BadRequestException. Otherwise, we should throw 500.